src/page-modules/departures/layout.tsx

```diff
@@ -128,6 +130,13 @@ function DeparturesLayout({
               options={['now', 'departBy']}
             />
           </div>
+
+          {geolocationError !== null && (
+            <div className={style.geolocationError}>
+              <MonoIcon icon="status/Info" />
+              <Typo.p textType="body__primary">{geolocationError}</Typo.p>
+            </div>
```

Comment:
This might be a message component at some point?

Reply:
Yeah, think this should be Message component of type `info`. Here I think webshop can be reused/copied directly?

Reply:
We already have this component (https://github.com/AtB-AS/planner-web/blob/main/src/modules/situations/situation-message-box.tsx#L101) actually. Should not be a problem to move the `MessageBox` component to the components folder and use it for other scenarios than situations or notices

Reply:
Extracted it to components and used it instead. Nice, good idea.


Didn't bother adding the column-spanning to get the message box to span the entire width, but maybe I should do that? Requires either adding a wrapper element with the correct styling, or make it possible to pass a className to the message box component. IMO it looks okay being only beneath the search inputs.

<img width="1016" alt="2023-11-28 at 19 36 09@2x" src="https://github.com/AtB-AS/planner-web/assets/14045515/588e51f6-56b4-4e42-86dc-08465e6430ec">

Reply:
Could wrap it inside a div that has col span yeah. Think that is the better approach. 🤔

Reply:
Will do